src/controller/CHIPDeviceController.cpp

```diff
@@ -2682,12 +2710,18 @@ void DeviceCommissioner::PerformCommissioningStep(DeviceProxy * proxy, Commissio
             readPaths[numberOfAttributes++] =
                 app::AttributePathParams(endpoint, IcdManagement::Id, IcdManagement::Attributes::FeatureMap::Id);
         }
+
         // Always read the active mode trigger hint attributes to notify users about it.
         readPaths[numberOfAttributes++] =
             app::AttributePathParams(endpoint, IcdManagement::Id, IcdManagement::Attributes::UserActiveModeTriggerHint::Id);
         readPaths[numberOfAttributes++] =
             app::AttributePathParams(endpoint, IcdManagement::Id, IcdManagement::Attributes::UserActiveModeTriggerInstruction::Id);
-
+        readPaths[numberOfAttributes++] =
```

Comment:
Please fix the security bug here.  At least it seems to be causing CI to fail....  Though the error messages are terrible.

Reply:
just fixed the array number..thanks